assets/js/admin/settings/index.js

```diff
@@ -161,18 +161,64 @@ const EDD_Settings = {
 			recaptureRemoteInstall();
 		});
 
-		// Email Summaries
-		const emailSummaryRecipient = $( 'select[name="edd_settings[email_summary_recipient]"]' ),
-			emailSummaryCustomRecipients = $( 'textarea[name="edd_settings[email_summary_custom_recipients]"]' );
-
-		// Toggle Email Recipient option
-		emailSummaryRecipient.on( 'change', function() {
-			if ( $( this ).val() === 'admin' ) {
-				emailSummaryCustomRecipients.prop( 'readonly', true );
-			} else {
-				emailSummaryCustomRecipients.prop( 'readonly', false );
-			}
-		} );
+		/**
+		 * Email Summaries
```

Comment:
I'm wondering if this code could be extracted into its own script file and enqueued only on the email summaries screen--it's all reasonable and needed code but it feels like it's ended up being a lot.

Reply:
Let's leave it in here, as it makes more sense. The only things that have their own are taxes currently and that is because it is built on backbone.